### PR TITLE
feature: add file

### PR DESCRIPTION
**Pull request by tanobe**

### CODE REVIEW

File: practice_with_YumemiWeather/Test.swift [hunk 1]
```diff
@@ -0,0 +1 @@
+import Foundation
```
Comment (tanobe2) [inline practice_with_YumemiWeather/Test.swift:1]: ?

**Review by tanobe2 — APPROVED** [review]

File: practice_with_YumemiWeather/Test.swift (hunk 1 above)
Comment (tanobe) [reply practice_with_YumemiWeather/Test.swift:1]: yea!, it is test!

Comment (tanobe2) [inline practice_with_YumemiWeather/Test.swift:1]: test

**Review by tanobe2 — APPROVED** [review]

File: practice_with_YumemiWeather/Test.swift (hunk 1 above)
Comment (tanobe) [reply practice_with_YumemiWeather/Test.swift:1]: !

**Pull request merged**